doc/employees.md

```diff
@@ -1 +1,605 @@
 # Employees Module
+Module | Description 
+--- | --- | --- | ---
+[Add](#add) | POST | Add New Data Employee
+[View](#view) | GET | View Data Employee 
+[Edit](#edit) | PUT | Edit Data Employee 
+[Delete](#delete) | DELETE | Delete Data Employee
+
+## <a name="add"></a>Add
+
+### Endpoint 
+POST /employees
+
+### Database
+![](./employees.png)
+
+for add, you need login with company_id from companies tables and branch_id from branches tables
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | *
+Accept | application/json
+Email | alamat@emailcdc.com
+
+### Request Payloads
+Name | Type | Example Value
+--- | --- | ---
+employee_kode | string | 20201201  
+employee_nama | string | ferry
+stall_code | string | Heavy Repair
+```
```

Comment:
- Where does the field of jabatan_id come from? 
- if user login is main dealer user, i think you need add branch_id in payload request. But if user login is branch user, you can grab branch_id from user login.